README.md

```diff
@@ -52,25 +52,39 @@ $ a11y index.html
 
 ## Options
 
-Query help:
+### Query help:
 
 ```sh
 $ a11y --help
 ```
 
-Customise viewport size:
+### Customise viewport size
+
+Type: String
+Default: 1024x768
 
 ```sh
 $ a11y --viewport-size=800x600
 ```
 
-Verbose mode:
+### Set a custom delay before capturing the page
+
+Type: Number (seconds)
+Default: 0
```

Comment:
Maybe default to 1 second to fix most problems with it not being able to capture JS changes right after document load.